### PR TITLE
Rework travis ci

### PR DESCRIPTION
**Pull request by spiderbiggen**
this PR should add a better integration with travis without changing core files of the sdk

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: java
 install: mvn -pl "!doc" install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
 script: 
-- java -cp environment/target/tygronenv-*-jar-with-dependencies.jar login.Login $TYGRONPWD $TYGRONMAIL
+- java -cp environment/target/tygronenv-*-jar-with-dependencies.jar login.Login $email $password
```
Comment (tvleest) [inline .travis.yml:4]: Did you switch email and password on purpose?

Comment (spiderbiggen) [inline .travis.yml:4]: yes mail should be before the password... I accidentaly did it wrong the first time, hence why it didn't work

Comment (tvleest) [inline .travis.yml:4]: Ah makes sense.

**Comment by tvleest** [comment]
Looks good to me.

**Comment by daanwerf** [comment]
Looks fine, I approve

**Comment by MichielvdBerg** [comment]
Looks good.

**Pull request merged**